Armadillo/src/main/java/com/scribd/armadillo/analytics/PlaybackActionTransmitter.kt

```diff
@@ -60,10 +61,8 @@ internal class PlaybackActionTransmitterImpl(private val stateProvider: StateSto
                 } catch (e: Throwable) {
                     // Don't throw exception if user SDK >= android 12 and it's ForegroundServiceStartNotAllowedException
                     // Exceptions kill emissions, but the player still works even after ForegroundServiceStartNotAllowedException
-                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
-                        if (e !is ForegroundServiceStartNotAllowedException) {
+                        if (hasSnowCone() && e !is ForegroundServiceStartNotAllowedException) {
```

Comment:
Nitpick: think this needs to out-dent

Reply:
Good catch, it also revealed a break in logic I added absent-mindedly